extensions.json

```
@@ -10,7 +10,8 @@
 		"davidanson.vscode-markdownlint",
 		"ms-vscode.powershell",
 		"charliermarsh.ruff",
-		"editorconfig.editorconfig"
+		"editorconfig.editorconfig",
+		"njpwerner.autodocstring",
```

I don't think terminating arrays with commas is valid JSON

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

```suggestion
		"njpwerner.autodocstring"
```

---

This is true of standard JSON. These files are not standard JSON though (standard JSON doesn't support comments, either), and they do support terminating arrays with commas. See, for example, https://github.com/microsoft/vscode/issues/102061#issuecomment-656770244
> However, in our VSCode settings files, trailing commas are accepted.

---

Since this is allowable syntax and makes diffs cleaner, I think we should use it.

---

Unless you're concerned about external parsers choking on it? But if that's a concern we shouldn't allow comments either.

---

makes sense to me